plutus-pab/test/full/Plutus/PAB/CliSpec.hs

```diff
@@ -153,6 +153,20 @@ startPab pabConfig = do
   void . async $ runWithOpts handler mc opts
   sleep 5
 
+
+-- | Make a config from the 'bumped' one that re-uses some of the services
+-- from the primary one (the ones we're not starting).
+secondaryConfig :: Config -> Config -> Config
+secondaryConfig primary other =
+  other { chainIndexConfig = chainIndexConfig primary
+        }
+
+startPrimaryPab :: Config -> IO ()
+startPrimaryPab = startPab allServices
+
+startSecondaryPab :: Config -> IO ()
+startSecondaryPab = startPab $ ForkCommands [ StartMockNode, MockWallet, PABWebserver ]
```

Comment:
I think we had a definition with the commands for client services at some point.

Maybe it's nicer to say `delete ChainIndex allServices`? Not sure...

Reply:
Yes, I think that's nicer, because then it semantically agrees with the `secondaryConfig` thing which only mentions the chain index; I'll switch it!